net/bitlbee/Makefile

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright (C) 2006-2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=bitlbee
+PKG_VERSION:=3.4.2
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://get.bitlbee.org/src/
+PKG_MD5SUM:=3b620bf80c35d673cd08311792ec96be
+
+PKG_MAINTAINER:=Nikil Mehta <nikil.mehta@gmail.com>
+PKG_LICENSE:=GPL-2.0
+PKG_LICENSE_FILES:=COPYING
+
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/nls.mk
+
+define Package/bitlbee
+  SECTION:=net
+  CATEGORY:=Network
+  SUBMENU:=Instant Messaging
+  TITLE:=An IRC to other chat networks gateway
+  URL:=http://www.bitlbee.org/
+  DEPENDS:=+glib2 +libevent +libgcrypt +libgnutls $(ICONV_DEPENDS) $(INTL_DEPENDS)
```

Comment:
Here's what the configure script reports when trying to use openssl; it looks like they really want bitlbee binaries to be distributed that link to gnutls.  Let me know what you think:

No detection code exists for OpenSSL. Make sure that you have a complete
installation of OpenSSL (including devel/header files) before reporting
compilation problems.

Also, keep in mind that the OpenSSL is, according to some people, not
completely GPL-compatible. Using GnuTLS is recommended and better supported
by us. However, on many BSD machines, OpenSSL can be considered part of the
operating system, which makes it GPL-compatible.

For more info, see: http://www.openssl.org/support/faq.html#LEGAL2
                    http://www.gnome.org/~markmc/openssl-and-the-gpl.html

Please note that distributing a BitlBee binary which links to OpenSSL is
probably illegal. If you want to create and distribute a binary BitlBee
package, you really should use GnuTLS instead.

Also, the OpenSSL license requires us to say this:
 *    "This product includes software developed by the OpenSSL Project
 *    for use in the OpenSSL Toolkit. (http://www.openssl.org/)"